### ISSUE TITLE
Suggestion: Analogue Stick <> DPad Switch

### ISSUE THREAD

**Issue by emuchicken** [blueminder/flycast-dojo#31]
Issue: When using a controller that doesn't have a set of both analogue and Dpad, each game needs to be set up in controller settings which may alienate some of your audience - usually beginners not wishing to change settings. 
Ppsspp has an option to flip the dpad and analogue controls instantly (axisswitch) by pushing just one button that is bound.
If this is possible in flycast, it would make a lot of people happy 😁

**Comment by dante3732** [comment]
Yep, need switches Dpad=left stick or right stick and lefts stick=dpad or right stick.
